<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modify</class>
 <widget class="QDialog" name="Modify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Song title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Artist name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Album name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="albumNameText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="artistNameText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="songNameText">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To set the cover, drop the image file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
